breakpad-symbols/src/sym_file/parser.rs

```diff
@@ -1,8 +1,16 @@
 // Copyright 2015 Ted Mielczarek. See the COPYRIGHT
 // file at the top-level directory of this distribution.
 
-use nom::IResult::*;
-use nom::*;
+use nom::branch::alt;
+use nom::bytes::complete::{tag, take_while};
+use nom::character::complete::{char, digit1, hex_digit1, multispace1};
+use nom::character::{is_digit, is_hex_digit};
+use nom::combinator::{map, map_res, opt};
+use nom::error::{Error, ErrorKind, ParseError};
+use nom::multi::many0;
+use nom::number::complete::hex_u32;
```

Comment:
hey nom, why the fuck do you have this function for exactly and only u32

Reply:
I know, right!?